src/Course/Compose.hs

```diff
@@ -16,8 +16,8 @@ newtype Compose f g a =
 -- Implement a Functor instance for Compose
 instance (Functor f, Functor g) =>
     Functor (Compose f g) where
-  (<$>) =
-    error "todo: Course.Compose (<$>)#instance (Compose f g)"
+  -- (a -> b) -> Compose f g a -> Compose f g b
```

Comment:
FYI: it is possible to add explicit type signatures to instance members with `InstanceSigs` extension